src/main/java/com/marginallyclever/makelangelo/turtle/Turtle.java

```diff
@@ -281,8 +281,8 @@ private void getBounds(Point2D top,Point2D bottom) {
 	public void scale(double sx, double sy) {
 		for( TurtleMove m : history ) {
 			switch(m.type) {
-			case TurtleMove.DRAW_LINE:
-			case TurtleMove.TRAVEL:
+			case DRAW_LINE:
```

Comment:
why isn't this `MovementType.DRAW_LINE`?

Reply:
it is the switch on an enum, the compiler know it is on `TurtleMove`, it can't be anything else, thus it can be written like this.